starlite/middleware/exceptions/middleware.py

```diff
@@ -35,7 +34,7 @@ def __init__(self, app: "ASGIApp", debug: bool, exception_handlers: "ExceptionHa
         self.debug = debug
 
     async def __call__(self, scope: "Scope", receive: "Receive", send: "Send") -> None:
-        """The middleware's ASGI-callable.
+        """ASGI-callable.
```

Comment:
```suggestion
        """ASGI callable.
```
No hyphen used in previous similar doc strings.